web-marketplace/src/pages/EcocreditsByAccount/EcocreditsByAccount.tsx

```diff
@@ -33,6 +34,7 @@ export const EcocreditsByAccount = (): JSX.Element => {
 
   const { address, party } = useProfileData();
   const { avatarImage, backgroundImage } = getUserImages({ party });
+  const isProfileNotFound = !address && !party;
```

Comment:
when the page's loading, we can see the "profile not found" page for a few ms
we should probably add a loader